.github/workflows/general.yml

```diff
@@ -45,6 +45,8 @@ jobs:
         key: ${{ runner.os }}-tools-${{ hashFiles('tools/**') }}
     - name: Make Mage
       run: make tools/bin/mage
+    - name: Auto-completion scripts
+      run: tools/bin/mage cli:autocomplete
```

Comment:
I'm fine with this if it runs very quick (i.e. below 10 seconds), if not - I don't think it belongs here

Reply:
This only takes a few seconds.

Reply:
I'd say we only need this in (snapshot) releases.

Reply:
I think it's quite beneficial to have this for Go builds as well